packages/react/src/components/form/Field/FieldCounter.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useRef, useState } from 'react';
+import { ValidationMessage } from '../../ValidationMessage';
+
+type CharacterCounterProps = {
+  /** Label template for when `maxCount` is exceeded
+   * @default '%d tegn for mye'
+   */
+  over?: string;
+  /** Label template for count
+   * @default '%d tegn igjen'
+   */
+  under?: string;
+  /** The maximum allowed characters. */
+  maxCount: number;
+};
+
+const label = (text: string, count: number) =>
+  text.replace('%d', Math.abs(count).toString());
+
+export const FieldCounter = ({
+  maxCount,
+  under = '%d tegn igjen',
+  over = '%d tegn for mye',
+}: CharacterCounterProps): JSX.Element => {
+  const [count, setCount] = useState(0);
+  const counterRef = useRef<HTMLSpanElement>(null);
+  const hasExceededLimit = count > maxCount;
+  const currentCount = maxCount - count;
+
+  useEffect(() => {
+    const onChange = (e: Event) => {
+      const element = e.target as HTMLInputElement | HTMLTextAreaElement;
+      setCount(element.value.length);
+    };
```

Comment:
This might be un-true, so maybe we can add a check? ☺️ 
```suggestion
    const onInput = ({ target }: InputEvent) => {
      if (target instanceof HTMLInputElement || target instanceof HTMLTextAreaElement)
        setCount(target.value.length);
    };
```

Reply:
Is this check maybe not redundant since we are listening in on an `InputEvent`, which are only thrown by input, textarea or select?

Reply:
I remember now. `InputEvent` type does also not work for `addEventListener` so thats why I did not use it. I'll update it with a better check as suggested :)

Reply:
:) and _any_ element can do `.dispatchEvent(new Event('input'))` just for fun 🙈 not that we should account for every possible modification, but here the check is quite easy :)